packages/server/lib/controllers/sync.controller.ts

```diff
@@ -159,17 +159,31 @@ class SyncController {
 
     public async getAllRecords(req: Request, res: Response, next: NextFunction) {
         try {
-            const { model, delta, limit, filter, cursor } = req.query;
+            const { model, delta, updated_after, updatedAfter, limit, filter, cursor, next_cursor } = req.query;
             const environmentId = getEnvironmentId(res);
             const connectionId = req.get('Connection-Id') as string;
             const providerConfigKey = req.get('Provider-Config-Key') as string;
 
+            if (updatedAfter) {
+                const error = new NangoError('incorrect_param', { incorrect: 'updatedAfter', correct: 'updated_after' });
+
+                errorManager.errResFromNangoErr(res, error);
+                return;
+            }
+
+            if (next_cursor) {
+                const error = new NangoError('incorrect_param', { incorrect: 'next_cursor', correct: 'cursor' });
```

Comment:
A customer recently shot themself in the foot with this: https://nangohq.slack.com/archives/C06GHQPLTAN/p1709405405531569